src/reducers/QuestionnaireReducers/questionList.json

```diff
@@ -43,43 +45,73 @@
             "2nd": 2,
             "3rd": 3, 
             "4th": 4,
-            "5+": 5
+            "5th+": 5
           }
         },
+        {
+          "id": "shirtSize",
+          "title": "What is your shirt size?",
+          "inputType": "multiSelect",
+          "choices": {
+            "XS": "xs",
+            "S": "s",
+            "M": "m",
+            "L": "l",
+            "XL": "xl",
+            "XXL": "xxl"
+          }
+        }
+      ]
+    },
+    {
+      "id": 1,
+      "name": "Demographics",
+      "questions":
+      [
         {          
-          "id": "city",
-          "title": "Which city will you be traveling from?",
-          "placeholder": "San Luis Obispo, CA",
-          "inputType": "textInput"
-        },
-        {          
-          "id": "diet",
-          "title": "Dietary Restrictions",
-          "inputType": "dropDown",
+          "id": "ethnicity",
+          "title": "What's your ethnicity?",
+          "inputType": "multiSelect",
           "choices": {
-            "None": 0, 
-            "Kosher": 1, 
-            "Gluten Free": 2, 
-            "Vegan": 3, 
-            "Vegetarian": 4
+            "White": "White", 
+            "Hispanic and Latino/a": "Hispanic and Latino/a", 
```

Comment:
What is this \a

Reply:
As per the google doc:
"
2. Demographics
   - Ethnicity*
      - White
      - Hispanic and Latino/a
      - Black or African American
      - Asian American
      - Native American/Alaska Natives
      - Native Hawaiian or Other Pacific Islander
      - Two or more races
      - Other
      - Decline to state
"

The "/a" part is to indicate "Latino" or "Latina". :)

Reply:
Alright well ship it!